ios/engine/KMEI/KeymanEngine/Classes/Model/KeymanPackage.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  KeymanPackage.swift
+//  KeymanEngine
+//
+//  Created by Jacob Bullock on 2/16/18.
+//  Copyright © 2018 SIL International. All rights reserved.
+//
+
+import Foundation
+
+public class KeymanPackage
+{
+  static private let kmpFile = "kmp.json"
+  public var sourceFolder: URL
+  private var keyboards: [KMPKeyboard]!
+
+  init(folder: URL) {
+    sourceFolder = folder
+  }
+  
+  public func parse(json: [String:AnyObject]) {
+    self.keyboards = []
+    
+    if let packagedKeyboards = json["keyboards"] as? [[String:AnyObject]] {
+      for keyboardJson in packagedKeyboards {
+        let keyboard = KMPKeyboard.init(kmp: self)
+        keyboard.parse(json: keyboardJson)
+        
+        if(keyboard.isValid) {
+          keyboards.append(keyboard)
+        }
+      }
+    }
+  }
+  
+  public func install() -> Int {
+    var count = 0
+    for keyboard in self.keyboards {
+      count += keyboard.installableKeyboards.count
+      keyboard.install()
+    }
+    
+    return count
+  }
+  
+  public func defaultInfoHtml() -> String {
+    var str = "Found Packages:<br/>"
```

Comment:
default info should display the package ID (based on the .kmp filename) instead of keyboard IDs.

Is the package version displayed somewhere too? Maybe on the title bar